apps/studio/src/features/editing-experience/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface MovedItem {
```

Comment:
nit: technically not moved yet

```suggestion
export interface PendingMoveResource {
```